frontend/lib/common.js

```diff
@@ -9,7 +9,10 @@ import getEnvVars from '../environment';
  * BASE("Products")...
  **/
 
-const { AIRTABLE_API_KEY, AIRTABLE_BASE_ID } = getEnvVars();
+const { AIRTABLE_API_KEY, AIRTABLE_BASE_ID, IMG_API_KEY } = getEnvVars();
 export const BASE = new Airtable({ apiKey: AIRTABLE_API_KEY }).base(
```

Comment:
For some reason, this is different than what's currently in master - changed it to match!